CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ BUG FIXES:
 * Fixed regression where `tofu` was not ignoring GPG signing key expiration by default ([822](https://github.com/opentofu/opentofu/issues/822))
   - Added experimental env variable to enforce checking of GPG signing key expiration `OPENTOFU_ENFORCE_GPG_EXPIRATION=true`
   - This is planned to default to `true` in a future release.
+* cloud: fixed a bug related to `workspaces` configuration using environment variables ([#814](https://github.com/opentofu/opentofu/issues/814))
```

Comment:
I think people may come along and hit `ctrl+F` and search for `TF_WORKSPACE` in our changelog if they did hit this issue, maybe we can be a bit more verbose here?

Maybe something like
```suggestion
* cloud: fixed a bug related to `workspaces` configuration not correctly recognizing TF_WORKSPACE env var ([#814](https://github.com/opentofu/opentofu/issues/814))
```

Reply:
@Yantrio thank you for suggestion James! I agree with it 👍🏻